src/web/app/pages-help/instructor-help-page/instructor-help-section/instructor-help-section.component.ts

```diff
@@ -0,0 +1,108 @@
+import {
+  AfterViewInit, Component, ElementRef, Input, OnChanges, OnInit, QueryList,
+  SimpleChanges, ViewChildren,
+} from '@angular/core';
+
+interface QuestionDetail {
+  tag: number;
+  keywords: String[];
+}
+
+/**
+ * Base section for instruction help page.
+ */
+@Component({
```

Comment:
If this class is not used as a component anywhere (just acting as a superclass), this directive is not necessary.

Reply:
Removed the directive! (and moved this file outside)